backend/grant-service/src/main/kotlin/grant/services/Applications.kt

```diff
@@ -502,6 +502,36 @@ class ApplicationService(
         )
     }
 
+    suspend fun transferApplication(
+        ctx: DBContext,
+        actor: Actor,
+        currentProject: String?,
+        applicationId: Long,
+        transferToProjectId: String
+    ) {
+        if (currentProject.isNullOrEmpty()) {
+            RPCException.fromStatusCode(HttpStatusCode.BadRequest, "Not in a project. No applications in Personal workspace")
+        }
+        if (projects.isAdminOfProject(currentProject!!, actor)) {
+            ctx.withSession { session ->
+                session
+                    .sendPreparedStatement(
+                        {
+                            setParameter("transfer", transferToProjectId)
+                            setParameter("applicationId", applicationId)
+                        },
+                        """
+                        UPDATE "grant".applications
+                        SET resources_owned_by = :transfer
+                        WHERE id = :applicationId
+                    """
+                    )
+            }
+        } else {
+            RPCException.fromStatusCode(HttpStatusCode.Forbidden)
```

Comment:
Missing `throw` keyword
```suggestion
            throw RPCException.fromStatusCode(HttpStatusCode.Forbidden)
```